plugins/kmeans.cc

```diff
@@ -197,7 +198,15 @@ run(const ProcedureRunConfig & run,
     if (!runProcConf.modelFileUrl.empty()) {
         try {
             Datacratic::makeUriDirectory(runProcConf.modelFileUrl.toString());
-            kmeans.save(runProcConf.modelFileUrl.toString());
+            Json::Value md;
```

Comment:
The problem is that people using saved k-means models were probably getting the wrong results from them, silently, because no record was made of which columns were provided.  Agreed that people who have saved models need to be made aware of it, but I don't think there are any in the wild ( @nicolaskruchten please verify).